rest/src/main/java/fi/otavanopisto/pyramus/rest/StudentRESTService.java

```diff
@@ -1731,7 +1738,7 @@ public Response getStudentMatriculationEligibility(@PathParam("ID") Long student
 
   @Path("/students/{ID:[0-9]*}/contactLogEntries")
   @POST
-  @RESTPermit(StudentContactLogEntryPermissions.CREATE_STUDENTCONTACTLOGENTRY)
+  @RESTPermit(handling = Handling.INLINE)
   public Response createStudentContactLogEntry(@PathParam("ID") Long id, fi.otavanopisto.pyramus.rest.model.StudentContactLogEntry entity) {
```

Comment:
Shouldn't this endpoint have the `ContactLogAccess access = studentController.resolveContactLogAccess(student);` check too